### PR TITLE
Milestone 2

### PR DESCRIPTION
**Pull request by Trictonicmp**
#### In this Mileston I:
Added methods to:
- [x] Users

    A method that returns the 3 most recent posts for a given user.

- [x] Posts

    A method that updates the posts counter for a user.
    A method which returns the 5 most recent comments for a given post.

- [x] Comments

    A method that updates the comments counter for a post.

- [x] Likes

    A method that updates the likes counter for a post.

### CODE REVIEW

File: app/models/post.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments, class_name: 'Comment'
+  has_many :likes, class_name: 'Like'
+  after_save :add_comment_count
```
Comment (uwadonat) [inline app/models/post.rb]: - There is no comment in this model. It should be `add_post_count`

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @Trictonicmp,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Descriptive pull request message ✔️
almoo

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- To remove unneeded lines in your readme file to make it more clean
Screenshot from 2022-08-12 09-14-18

- Make sure this `MIT` link is working
Screenshot from 2022-08-12 09-13-57



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Trictonicmp** [comment]
Hi @uwadonat!!
Thank you for your feedback, I made the changes requested, I'll ask for another review!

**Comment by omar-labana** [comment]
# **Approved ✔️**

Hi @Trictonicmp

Congratulations on completing this project! 👏🏼 🎉

You have correctly implemented all of the requirements📃 correctly and without any errors⚠️ or bugs🪲.

Cheers and good luck in your next milestones! 🍀

Feel free to leave any questions or comments in the PR thread(don't forget to tag me 😄) if something is not 100% clear.

---
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**